import_export_extensions/admin/model_admins/export_job_admin.py

```diff
@@ -116,21 +116,19 @@ def export_job_progress_view(
             percent = int(100 / total * current)
 
         response_data.update(
-            dict(
-                state=job_progress["state"],
-                percent=percent,
-                total=total,
-                current=current,
-            ),
+            state=job_progress["state"],
+            percent=percent,
+            total=total,
+            current=current,
         )
         return JsonResponse(response_data)
 
-    def get_readonly_fields(self, request, obj=None):
-        """Return readonly fields.
-
-        Some fields are editable for new ExportJob.
-
-        """
+    def get_readonly_fields(
+        self,
+        request: WSGIRequest,
+        obj: models.ExportJob,
+    ):
+        """Return readonly fields."""
```

Comment:
So why you need to override this method instead of using `readonly_fields` attribute?